samples/boards/espressif/deep_sleep/Kconfig

```diff
@@ -23,4 +23,11 @@ config EXAMPLE_GPIO_WAKEUP
 	  as wake-up source. Be aware that when low level is used to trigger
 	  wake-up, an external pull-up resistance is necessary.
 
+config EXAMPLE_WAKEUP_TIME_SEC
```

Comment:
Why is it necessary to tinker with a timeout? I see it is applied only once.

Reply:
So that we can speed up CI tests. I see I could have just converted the 5s to 1s in original variable but I'd rather keep as is.